EnglishFilter/sections/specific.txt

```diff
@@ -204,6 +204,7 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 !
 smallseotools.com##.top_b_home
 smallseotools.com##div[id^="stop_it_"]
+wordcounter.icu##.interstitial-page > nav#mainNav ~ *
```

Comment:
What about, making this rule more specific?
For example:
`wordcounter.icu##.interstitial-page > nav#mainNav ~ iframe`